node4.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+module.exports = {
+  extends: require.resolve('./lib/base-node'),
+  rules: {
+    'node/no-unsupported-features': ['error', { version: 4 }],
```

Comment:
Does the rules object here combine with the object being extended? That is, does this `.js` file end up making the `'node/no-missing-require'` statement from `base-node.js`?